README.md

```diff
@@ -34,26 +34,22 @@ Start the database with:
 docker compose up -d
 ```
 
-You can access the database inside the Docker container with:
+The database is initialized but not populated. To populate it you can use the test data:
 
 ```
-docker exec -it postgres psql
+psql -h localhost -U postgres -d tac -f './server/src/dbUtils/populate_test.sql'
```

Comment:
Maybe mention the password (`PGPASSWORD=postgres`) somewhere.

Reply:
Good idea